reference/forms/types/percent.rst

```diff
@@ -54,6 +56,20 @@ scale
 By default, the input numbers are rounded. To allow for more decimal places,
 use this option.
 
+symbol
+~~~~~~
+
+**type**: ``boolean`` **default**: ``true``
+
+.. versionadded:: 4.3
+
+    The ``symbol`` option was introduced in Symfony 4.3.
+
+By default, fields are rendered with a percentage sign "`%`" after the input.
+Setting the value to ``false`` will not display the percentage sign.
+Setting the value to a `string` (.i.e. "`‱`") it will be shown instead of
```

Comment:
Imo we should write it like this:
```diff
- Setting the value to a string (e.g. ``'‱'``) it will be shown instead of
+ Setting the value to a string (e.g. ``‱``) it will be shown instead of
```

WDYT @javiereguiluz ?